home/models/models.py

```diff
@@ -18,6 +19,7 @@ class UserProfile(mdls.Model):
     change_pwd = mdls.BooleanField(default=False)
     date_of_birth = mdls.DateField(default="1901-01-01")
     salesforce_id = mdls.CharField(default="xxxxxx19010101", max_length=14)
+    in_classroom = mdls.BooleanField(default=False)
```

Comment:
What is this going to be used for?

Reply:
Taken care of in 35858ae5613e038813d6ac6c2383bc679359b3dd